site/content/docs/03-run-time.md

```diff
@@ -162,6 +162,22 @@ Like lifecycle functions, this must be called during component initialisation.
 
 > Context is not inherently reactive. If you need reactive values in context then you can pass a store into context, which *will* be reactive.
```

Comment:
```suggestion
> Context is not inherently reactive. If you need reactive values in context then you can [pass a store into context](#reactive-contextapi), which *will* be reactive.
```

Reply:
@windyclory Move it to between `getAllContexts` and `createEventDispatcher` and then add a link here is what I meant.

Apart from that, could you also add the missing semicolons in the tutorial files? Not only for consistency with the rest of the tutorial but not having them also seems to be messing with syntax highlighting in my editor.

Reply:
> Move it to between getAllContexts and createEventDispatcher and then add a link here is what I meant.

I don't want to add it between `getAllContexts` and `createEventDispatcher` because it's better to show it after the ContextAPI reactivity alert so the user that is looking for the reactive version be able to find it easily. And it's basically related to setContext but in a reactive way.

Do the link thing yourself because I don't know much about it.